docs/recipes/multi-debit-credit-transfers.md

```diff
@@ -42,9 +42,9 @@ This example debits multiple accounts and credits a single account. It uses the
 
 ## Many-to-Many Transfers
 
-Transactions with multiple debits and multiple credits are a bit more tricky.
+Transactions with multiple debits and multiple credits are a bit more involved (but you got this!).
 
-To simplify the math, we can use a temporary "control account" as an intermediary.
+This is where the accounting concept of a Control Account comes in handy. We can use this as a temporary intermediary account, as illustrated below.
```

Comment:
```suggestion
This is where the accounting concept of a Control Account comes in handy. We can use this as an intermediary account, as illustrated below.
```

To remove "temporary", since the control account remains. It's more that the financial state transition goes in/out or transitions through immediately.